apiserver/root.go

```diff
@@ -649,8 +651,9 @@ func GetTLS(certPath, keyPath string) error {
 
 }
 
-// generate a self signed cert and the pgo.tls Secret to hold it
-func generateTLS(certPath, keyPath string) error {
+// generateTLSCert generates a self signed cert and the stores it in both
```

Comment:
`the` -> `then`

Reply:
thanks, corrected in a local commit - holding for any other requested changes.